### PR TITLE
[BLOCK] run owncloud 10 legacy API tests in CI

### PR DESCRIPTION
**Pull request by individual-it**
part of #884
on top of #896

### CODE REVIEW

File: drone/oc-integration-tests/storage-publiclink.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This storage.toml config file will start a reva service that:
+[shared]
+jwt_secret = "Pive-Fumkiu4"
+gatewaysvc = "localhost:19000"
+
+[grpc]
+address = "0.0.0.0:13000"
+
+# This is a storage proider that grants direct acces to the wrapped storage
```
Comment (phil-davis) [inline drone/oc-integration-tests/storage-publiclink.toml:9]:
```suggestion
# This is a storage provider that grants direct access to the wrapped storage
```

This keeps getting copy-pasted :)

File: drone/oc-integration-tests/start.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+REVAD=~/www/reva/cmd/revad/revad
```
Comment (phil-davis) [inline drone/oc-integration-tests/start.sh:1]: This does not seem to be used. The commands are inline in the drone pipeline(s).

Comment (individual-it) [reply drone/oc-integration-tests/start.sh]: just a local file that is not needed in CI

**Comment by labkode** [comment]
@individual-it please rebase

**Comment by phil-davis** [comment]
I rebased and resolved conflicts. IMO I got it right! Let's look in detail at the CI tests that run.

**Comment by labkode** [comment]
@phil-davis some step has failed.

**Comment by phil-davis** [comment]
hmmm - no drone started after the last commit was pushed???

**Comment by phil-davis** [comment]
Note: this needs to wait until the upstream core PR https://github.com/owncloud/core/pull/37624 is finalized and merged. Then I  can adjust the core branch+commit in `.drone.star` here to specify the final core master commit.

**Comment by phil-davis** [comment]
Trying to get this correct now. I am getting:
```
git push
remote: Permission to cs3org/reva.git denied to phil-davis.
fatal: unable to access 'https://github.com/cs3org/reva.git/': The requested URL returned error: 403
```

Did I lose access somehow in the last hour?

**Comment by phil-davis** [comment]
See PR #927 which I made from my own fork, since I seem to have lost write access to this repo.

**Pull request closed without merging**